package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "spacex",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.8.5",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/user-event": "^13.5.0",
+    "bootstrap": "^5.2.0",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0",
+    "react-bootstrap": "^2.5.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.0.2",
+    "react-router": "^6.3.0",
+    "react-router-dom": "^6.3.0",
+    "react-scripts": "5.0.1",
+    "redux-thunk": "^2.4.1",
```

Comment:
- [ ] All the required packages are installed except Redux Logger, please install [Redux Logger](https://www.npmjs.com/package/redux-logger) as specified in the project requirement.